pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -159,6 +161,9 @@ stdenv.mkDerivation rec {
   NIX_CFLAGS_COMPILE = "-DG_ENABLE_DEBUG -DG_DISABLE_CAST_CHECKS";
 
   postPatch = ''
+    # See https://github.com/NixOS/nixpkgs/issues/132259
+    sed -i "0,/x11_enabled = false/s///" meson.build
```

Comment:
Can we use substituteInPlace to at least log a warning when this fails?

Reply:
alright. substituteInPlace will also remove the flag for windows but that probably wont matter https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/meson.build#L164